### ISSUE TITLE
no man page, no config files

### ISSUE THREAD

**Issue by dhurler** [teejee2008/ukuu#4]
I could not find a configuration file for ukuu anywhere on my computer after installing it nor did an inspection of the .deb file reveal anything. Ukuu would be much better if there were means to configure how it works and if there was a man page for it. I would propose allowing the user to set up which kernel repositories it can use to include non-Ubuntu and kernels other than "mainstream" when using the Ubuntu kernels. There probably should be separate configuration files for the command line and it's GTK app as well as a file that holds all of the kernel repositories ukuu accesses.

**Comment by teejee2008** [comment]
User specific config files are created in $HOME/.config/ukuu.json

There are no plans to add support for other kernel types.

**Comment by dhurler** [comment]
I noticed these files after I changed settings in the GUI app but they
weren't present until I made these changes.

It's unfortunate you do not want to add other kernel types because I think
there's a great need for a generic GUI kernel installation/update/uninstall
utility and it appears your application is the only viable candidate out
there that could fill this role.

I think adding more transparent options for how the app schedules updates
would be a nice since users may prefer to only use this app manually.
Thanks.

On Apr 11, 2017 9:00 PM, "Tony George" <notifications@github.com> wrote:

User specific config files are created in $HOME/.config/ukuu.json

There are no plans to add support for other kernel types.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/teejee2008/ukuu/issues/4#issuecomment-293468054>, or mute
the thread
